RTN-004.tex

```diff
@@ -53,7 +57,320 @@
+
+\begin{table}[!h]
+\centering
+\caption{Projected DP0 delegate application process timeline.}\label{tab:selection_timeline}
+\begin{tabular}{lllll}
+\hline
+Stage & \% (of 300) & Applications Due & Delegates Notified & Data Available \\
+\hline \hline
+DP0.1 & 75\%    &  Apr 30 2021   &  May 31 2021  &  Jun 30 2021 \\
+DP0.2 & 100\%  &  Jan 31 2022   &  Feb 28 2022  &  Mar 31 2022 \\
+\hline
+\end{tabular}
+\end{table}
+
+
+\section{Resources for DP0 Delegates}\label{sec:res}
+
+\subsection{Policies for RSP Accounts}\label{ssec:res_pol}
+
+For the duration of DP0, the RSP will be installed in the Interim Data Facility (IDF) deployed on the Google Cloud Platform.
+More information about the IDF and its installation will be available elsewhere.
+Access to the RSP and the DP0 data products and services are limited to data rights holders, and all DP0 delegates must abide by the data rights policies in \citeds{RDO-013}.
+
+All DP0 delegates will be asked to abide by specific policies related to account authentication, security, use, and deactivation.
+The full details for these policies will be provided to delegates prior to account creation. 
+
+Delegates may not share accounts or passwords, but will be free (and encouraged!) to provide tours of the RSP or live demonstrations of their work to individuals who are not DP0 delegates.
+Account usage policies might include, e.g., best practices for starting and stopping servers, understanding that computational processes which overburden the system might be curtailed, abiding by storage capacity quotas, and following on- and off-boarding steps.
+
+As the total number of accounts will be limited to 300, if applications are oversubscribed then delegates who don't appear to be using their accounts might be periodically contacted and asked to consider voluntarily deactivating to make way for others.
+In the case of a high oversubscription for DP0 participation, account holders who do not respond to a sustained number of such inquiries might have their access suspended to make way for others.
+
+However, delegates who are given accounts at the start of DP0.1 but who do not have the work-hours to start using them for a few months (due to prior commitments, teaching loads, etc.) will be accommodated. 
+Active, responsive delegates who are given accounts for DP0.1 may keep them, and will not need to reapply for DP0.2.
+
+Policies related to account suspension will be clarified for DP0 delegates during on-boarding.
+Note that this is a temporary measure for the Data Previews, and not how accounts will work during Operations.
+
+\subsection{Support for Delegates}\label{ssec:res_sup}
+
+It is recognized that on-boarding resources and organizational infrastructure provided by Rubin Observatory will be necessary to enable equitable access and benefits for all delegates, including novice science platform users (Section \ref{ssec:sel_equity}). 
```

Comment:
Up to you, but I would strongly suggest that you add a sentence along the lines of "This is a time where we are still building up our support processes and staffing, and some patience might be required". Unless you're really thinking everything is going to be awesome on Day 1...

Reply:
Added "However, potential DP0 delegates should be aware that the Rubin Observatory pre-operations team is still building their support resources and infrastructure, and some patience might be required."